general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -109,6 +109,17 @@ def request() -> requests.Response:
 
         return self._make_request(request).json()
 
+    def get_superstaq_version(self) -> Dict[str, Optional[str]]:
+        """Gets SuperstaQ version from response header
```

Comment:
nit: docstring format, add a newline
```suggestion
        """Gets SuperstaQ version from response header

```